action.yml

```diff
@@ -0,0 +1,101 @@
+name: "Sourcery GitHub Action"
+
+branding:
+  icon: 'box'
+  color: 'orange'
+
+description: "A GitHub Action for running Sourcery."
+
+inputs:
+  token:
+    description: "The Sourcery token for logging in."
+    required: true
+  target:
+    description: "File(s) or directory(ies) to review."
+    required: false
+    default: "."
+  version:
+    description: "Sourcery CLI version, e.g. '0.12.10'. If empty, default to the latest version."
+    required: false
+    default: ""
+  verbose:
+    description: "Verbose output with explanation and code snippets. Either 'true' or 'false'."
+    required: false
+    default: "true"
+  check:
+    description: "Fail job if issues are found. Either 'true' or 'false'."
+    required: false
+    default: "true"
+  in_place:
+    description: "Change files in place. Either 'true' or 'false'."
+    required: false
+    default: "false"
+  config:
+    description: "Location of the Sourcery YAML configuration file."
```

Comment:
🤓 nitpick: We could add here some additional info.

* If its empty only the project's `.sourcery.yaml` file is used.
* Perhaps even a link to the GPSG yaml

Reply:
I don't think that this explanation fits here. The description for each input should be as concise as possible (this is what I see in other GH Actions). I believe that the official docs are a better place to provide that additional information. I created an issue for us to keep track of this idea: https://github.com/sourcery-ai/core/issues/2031.